content/posts/2024-07-30_crowdstrike-risk-at-the-edge.md

```diff
@@ -0,0 +1,39 @@
++++
+date = 2024-07-30T00:37:35+02:00
+title = "Risk assessment must happen at the sharp end"
+description = "I think the Crowdstrike is a great example of centralised risk management gone wrong."
+tags = ["security", "risk"]
+slug = "centralised-risk-management"
++++
+Today I wanted to write something about [the recent Crowdstrike Falcon debacle][crowdstrike-debacle] and about one contributing factor, namely _centralised risk management_.
+
+[crowdstrike-debacle]: https://en.wikipedia.org/wiki/2024_CrowdStrike_incident
+
+## The importance of local risk assessment
+
+I recently watched [the trailer][safety-differently-trailer] for the movie ["Safety Differently"][safety-differently] by Sidney Dekker. One of the things it talks about is that **the farther you are from a system, the worse your risk management controls (you come up with) will be**. Or to put it another way, the grassroots operators of a system are usually the best people to come up with controls and defend against catastrophes.
+
+[safety-differently-trailer]: https://www.youtube.com/watch?v=IEYN38nir_w
+[safety-differently]: https://www.youtube.com/watch?v=EeIucLnEa24
+
+For example, if you are a security & risk department far away from your developers, the controls you put in place for the developer's software will likely not reduce risk the way you expect. You might [add security controls that make operational risk worse][operational-risk], etc. Instead of enforcing specific controls on all developers, a much better approach is to give the requirement and work _together_ with the developers to come up with a solution. Don't do "we must use nginx for TLS". Instead, tell them that transport encryption is a must and they get to pick their favorite HTTP server that works best for them.
```

Comment:
Relevant? Remove? Or move to a completely separate section at the end of the article and also mention monorepos and dependencies?

Reply:
Will write about monorepos in some other post.